### PR TITLE
Added Translation test & services

### PR DESCRIPTION
**Pull request by Steveb-p**
| Question                       | Answer                                                |
|--------------------------------|-------------------------------------------------------|
| **JIRA issue**                 | [IBX-XXXXX](https://issues.ibexa.co/browse/IBX-XXXXX) |
| **Type**                       | feature/bug/improvement                               |
| **Target eZ Platform version** | `v4.5` - please update `x` accordingly                |
| **BC breaks**                  | yes/no                                                |
| **Doc needed**                 | yes/no                                                |

This PR adds the ability to perform translation checks.

#### Checklist:
- [x] Provided PR description.
- [x] Tested the solution manually.
- [x] Provided automated test coverage.
- [x] Checked that target branch is set correctly (master for features, the oldest supported for bugs).
- [x] Asked for a review (ping `@ibexa/engineering`).

### CODE REVIEW

File: src/contracts/Translation/AbstractTranslationCase.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+use JMS\TranslationBundle\Model\Message;
+
+abstract class AbstractTranslationCase extends IbexaKernelTestCase
+{
+    abstract protected function getConfigName(): string;
+
+    final public function testTranslation(): void
+    {
+        $facade = $this->getTranslationService();
+        $changeset = $facade->getChangeSet($this->getConfigName());
```
Comment (alongosz) [inline src/contracts/Translation/AbstractTranslationCase.php]: How about making it a data provider? That way you could also test more of them if needed.

File: src/contracts/Translation/Translation.php [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+namespace Ibexa\Contracts\Test\Core\Translation;
+
+use JMS\TranslationBundle\Translation\Comparison\ChangeSet;
+use JMS\TranslationBundle\Translation\ConfigFactory;
+use JMS\TranslationBundle\Translation\Updater;
+use Twig\Environment;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
+
+final class Translation
```
Comment (alongosz) [inline src/contracts/Translation/Translation.php]: How about `TranslationService`, so it doesn't look like a `Value`?

File: src/contracts/Translation/AbstractTranslationCase.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        }
+
+        self::assertCount(0, array_merge($addedMessages, $deletedMessages), $message);
+    }
+
+    private function getTranslationService(): Translation
+    {
+        $service = self::getContainer()->get(Translation::class);
+        if (!$service instanceof Translation) {
+            throw new \LogicException(sprintf(
```
Comment (konradoboza) [inline src/contracts/Translation/AbstractTranslationCase.php]:
```suggestion
            throw new LogicException(sprintf(
```

File: src/contracts/Translation/Translation.php (hunk 2 above)
Comment (konradoboza) [reply src/contracts/Translation/Translation.php]: +1 given this line https://github.com/ibexa/test-core/pull/4/files#diff-91a06b220f2c897d192c7aa0455b5b1e8f7f547450ae0835d8af052567ed79c8R64.

**Review by konradoboza — APPROVED** [review]

**Review by alongosz — APPROVED** [review]

**Review by mikadamczyk — APPROVED** [review]

**Pull request merged**